mock/py/mockbuild/buildroot.py

```diff
@@ -466,6 +468,9 @@ def _fixup_build_user(self):
     @traceLog()
     @noop_in_bootstrap
     def _make_build_user(self):
+        self.shadow_utils.create_user("pesign", copy_from_host=True)
+        self.shadow_utils.create_user("random-user")
```

Comment:
This part likely needs to be done elsewhere and needs to be configurable!  This is just a dummy preview for experiments.